README.md

```diff
@@ -518,6 +538,17 @@ both repositories. This is the first time they have needed to be used since the
 repositories' creation at the beginning of this demo. Root keys should be
 kept offline, as discussed in
 the [Uptane Deployment Considerations document](https://docs.google.com/document/d/17wOs-T7mugwte5_Dt-KLGMsp-3_yAARejpFmrAMefSE/edit?usp=sharing).
+If trust in a key needs to be revoked, the keys at a level above it (or any
+level above that) can be used.
+In the case of the top-level roles (like Timestamp, Snapshot, and Targets here),
+that falls to Root to do. In the case of delegated Targets roles, the role
+that delegates would do the revocation. So, for example, if you delegate
+the ability to sign for firmware from a given vendor to the security team from
+that vendor, and that team delegates in turn to a particular release manager,
+and that release manager's YubiKey falls into the sewers, the security team
```

Comment:
Oh, no.  The sewers?  Good, better than discovering that you have a vulnerable authentication device.

Reply:
Variety is the spice of security.